tests/spec/fs.open.spec.js

```diff
@@ -136,4 +136,13 @@ describe('fs.open', function() {
       });
     });
   });
+
+  it('should return an error when flag in invalid', function(done) { 
+    var fs = util.fs();
+
+    expect(fs.open('/myfile', 'abcd', function(){
```

Comment:
Usually you can't pass a function that will throw like this, you have to wrap it in a function that the test runner can call:

```js
expect(function() {
  fs.open('/myfile', 'abcd', function(){});
}).to.throw('flags is not valid');
```

It's sort of like having a fuse on a bomb: before you set a bomb off, you need to be far away.  By wrapping the function in an outer function, you can safely call the outer function, knowing that it will likely throw.